README.md

```diff
@@ -512,6 +518,27 @@ The `React\Promise\PromisorInterface` provides a common interface for objects
 that provide a promise. `React\Promise\Deferred` implements it, but since it
 is part of the public API anyone can implement it.
 
+### AsyncInterop\Promise compatibility
+
+The [`PromiseInterface`](#promiseinterface) extends the
+[`AsyncInterop\Promise`](https://github.com/async-interop/promise) interface and
+thereby all promise implementations from React/Promise implement its `when()`
+method.
+
+The `AsyncInterop\Promise` interface is intended for interoperability with other
+libraries and their combinator and conversion functions.
+
+Note, that although the interface is named *Promise*, it has nothing to do with
+promises commonly known from the
```

Comment:
@clue I understand your point, but I'm not sure whether it really helps consumers to understand it. They'll just think _"Why the hell is it called promise if it's something entirely different?!"_ and be more confused than before. We had a really long discussion about the naming in https://github.com/async-interop/promise/issues/4 and another one to switch back to `Promise`, but I can't find that one right now. The reason behind it is that we kept calling it `Promise` while talking about it, because it's just that, just with a different consumer API.